Makefile

```diff
@@ -40,34 +51,67 @@ ALL_SRCS = \
 	test_SimulationIO.cpp
 PYTHON_EXE = _H5.so _RegionCalculus.so _SimulationIO.so
 ALL_EXE = \
-	$(PYTHON_EXE) \
-	benchmark convert-carpet-output list example \
+	sio-benchmark sio-convert-carpet-output sio-list sio-example \
 	test_RegionCalculus test_SimulationIO
 
-HDF5_DIR = /opt/local
-HDF5_CPPFLAGS = -I$(HDF5_DIR)/include
-HDF5_CXXFLAGS =
-HDF5_LDFLAGS = -L$(HDF5_DIR)/lib -Wl,-rpath,$(HDF5_DIR)/lib
-HDF5_LIBS = -lhdf5_cpp -lhdf5
-
-MPI_DIR = /opt/local
-MPI_CPPFLAGS = -I$(MPI_DIR)/include/openmpi-gcc5
-MPI_CXXFLAGS =
-MPI_LDFLAGS = -L$(MPI_DIR)/lib/openmpi-gcc5 -Wl,-rpath,$(MPI_DIR)/lib/openmpi-gcc5
-MPI_LIBS = -lmpi_cxx -lmpi
-
-PYTHON_DIR = /opt/local/Library/Frameworks/Python.framework/Versions/2.7
-PYTHON_CPPFLAGS = -I$(PYTHON_DIR)/include/python2.7
-PYTHON_CXXFLAGS =
-PYTHON_LDFLAGS = -L$(PYTHON_DIR)/lib -Wl,-rpath,$(PYTHON_DIR)/lib
-PYTHON_LIBS = -lpython2.7
-
-GTEST_DIR = googletest-release-1.7.0
+HDF5_DIR ?= $(DEFAULT_HDF5_DIR)
```

Comment:
I will leave the decision what to do and when up to you for the release branch. You can e.g. ignore all my comments. We should clean things up later. Let me know whether I should just merge it.